modules/statisticsInterface.js

```diff
@@ -138,98 +138,98 @@ function statisticsInterface() {
+          if (counter[dataForLS[key].day] == null) {
+            counter[dataForLS[key].day] = 0;
+          }
+          counter[dataForLS[key].day]++;
+        }
+      }
+
+      let tableForStat = document.createElement(`div`);
+
+      let tableString = `
           <table>
               <tr>
                   <td>День</td>
                   <td>Штуки</td>
               </tr>
           `;
 
-            for (const key in counter) {
-                tableString =
-                    tableString +
-                    `
+      for (const key in counter) {
+        tableString =
+          tableString +
+          `
               <tr>
               <td>${key}</td>
               <td>${counter[key]}</td>
           </tr>`
-            }
-            tableString = tableString + `</table>`
-            tableForStat.innerHTML = tableString
-            tableForStat.style.color = `#FFF`
-            tableForStat.style.position = `absolute`
-            tableForStat.style.margin = `30px 10px 10px 10px`
-            document.querySelector("#b-footer > div > div.b-copyright").before(tableForStat)
-
-        }
+      }
+      tableString = tableString + `</table>`
+      tableForStat.innerHTML = tableString
+      tableForStat.style.color = `#FFF`
+      tableForStat.style.position = `absolute`
+      tableForStat.style.margin = `30px 10px 10px 10px`
+      document.querySelector("#b-footer > div > div.b-copyright").before(tableForStat)
 
     }
 
-    function setSettings() {
-        let elem = document.querySelector("#settings")
-        elem.addEventListener("click", () => {
-            let preSet = {}
+  }
 
-            for (let i = 0; i < document.querySelectorAll("#settings > input[type=checkbox]").length; i++) {
-                let x = document.querySelectorAll("#settings > input[type=checkbox]")[i]
-                if (x.checked) {
-                    preSet[x.name] = "checked"
-                } else {
-                    preSet[x.name] = ""
-                }
+  function setSettings() {
+    let elem = document.querySelector("#settings")
+    elem.addEventListener("click", () => {
+      let preSet = {}
 
-            }
-            preSet.usersSS = []
-            for (let i = 0; i < document.querySelectorAll("#settings > div > div > label").length; i++) {
-                let ss = document.querySelectorAll("#settings > div > div > label > input[type=checkbox]")[i]
-                if (ss.checked) {
-                    preSet.usersSS.push(i + 1)
-                }
-            }
+      for (let i = 0; i < document.querySelectorAll("#settings > input[type=checkbox]").length; i++) {
+        let x = document.querySelectorAll("#settings > input[type=checkbox]")[i]
+        if (x.checked) {
+          preSet[x.name] = "checked"
+        } else {
+          preSet[x.name] = ""
+        }
 
-            localStorage.setItem(`usersSettings`, JSON.stringify(preSet))
-        })
+      }
+      preSet.usersSS = []
```

Comment:
too long, i believe you can reduce it with the help of methods of js and ternary operators ;)